huey/contrib/djhuey/management/commands/run_huey.py

```diff
@@ -67,8 +65,6 @@ def add_arguments(self, parser):
                     short = '-V'
                 if 'type' in kwargs:
                     kwargs['type'] = self._type_map[kwargs['type']]
-                if kwargs.get('default') is not True:
-                    kwargs.pop('default', None)
```

Comment:
Unless the defaults are true (this is why `utc` and `no_periodic` are not used).
It's not worth having the defaults set in the optparser in the first place at all, which is why I suggested to remove them.